package.json

```diff
@@ -6,6 +6,7 @@
   "repository": "silverwind/dnsz",
   "license": "BSD-2-Clause",
   "scripts": {
+    "build": "make build-typings",
```

Comment:
Better to add a `build` make target instead and have the `test` target depend on it. Thought I generally would prefer to not have a build step in first place and just have a static `.d.ts` file.

Reply:
Removed the build scripts and make d.ts like manually written.